spec/models/concerns/metadata_sequence_number_spec.rb

```diff
@@ -5,27 +5,15 @@
     Class.new do
       def self.validates(*args)
       end
-
-      def self.before_validation(*args)
-      end
     end
   end
 
   it 'sets up validations for the metadata_sequence_number field' do
     expect(test_class).to receive(:validates).with(
-      :metadata_sequence_number, presence: true, uniqueness: true,
-                                 numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+      :metadata_sequence_number, uniqueness: { allow_nil: true }, numericality: {
+        only_integer: true, greater_than_or_equal_to: 0, allow_nil: true
+      }
     )
```

Comment:
should the `metadata_sequence_number` be nil?  Or maybe the `nil` is required since it's set by the PG function?

Reply:
Yeah attempting to set it in Rails messed up the default, it has to be left unset in the app.